docs/modules/ROOT/pages/how-tos/testing/integration/debugging.adoc

```diff
@@ -0,0 +1,44 @@
+= Debugging Integration Tests
+
+In the event of an integration test scenario failing, there are a number of steps to take in order to debug the issue before rerunning the test in question. These steps are outlined below.
+
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before attempting to debug the failed test.
+
+- You have access to the console in order to view the logs of the failed test.
```

Comment:
I think that it has been called the `{ProductName} UI` instead of the `console`.

Reply:
Thanks!